l10n/en/mozorg/try-picture-in-picture.ftl

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file needs to be added to a config, either pontoon.toml, vendor.toml, or both.